src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php

```diff
@@ -260,6 +265,21 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
                 $domainMessagesCount = \count($list);
 
+                if ($sort = $input->getOption('sort')) {
+                    $sort = strtolower($sort);
+                    if (!\in_array($sort, self::SORT_ORDERS, true)) {
+                        $errorIo->error(['Wrong sort order', 'Supported formats are: '.implode(', ', self::SORT_ORDERS).'.']);
+
+                        return 1;
+                    }
+
+                    if (self::DESC === $sort) {
+                        rsort($list);
+                    } else {
```

Comment:
is the else really needed ? If the sort option is wrong it should not come here, I think we can drop the else.

Reply:
how come? sort for asc order and rsort for reverse. or i misunderstood your comment?

Reply:
The `else` looks correct and necessary to me.